proposals/artifacts.md

```diff
@@ -0,0 +1,58 @@
+# OCI artifact project proposal
+
+## Abstract
+Container registries, implementing the [distribution-spec][distribution-spec], provide reliable, highly scalable, secured storage services for container images. Customers either use a cloud provider implementation, vendor implementations, or instance the open source implementation of distribution. They configure security and networking to assure the images in the registry are locked down and accessible by the resources required. Cloud providers and vendors often provide additional values atop their registry implementations from security to productivity features. 
+
+Applications and services typically require additional artifacts to deploy and manage, including [helm](https://helm.sh) for deployment and [Open Policy Agent (OPA)](https://github.com/open-policy-agent/opa/issues/1413) for policy enforcement.
```

Comment:
I think this should refer to the artefact instance, rather than the software. ie.

```diff
- Helm
+ Helm Charts
```

```diff
- Open Policy Agent
+ Open Policy Agent bundles
```

Reply:
/s/manage/manage container images/